### ISSUE TITLE
Inclusion in ansible-modules-extras

### ISSUE THREAD

**Issue by ryansb** [pjodouin/ansible-lambda#10]
Hello, I'm a member of the Ansible team, and (as you know) we currently don't have support for Lambda included. I see your code is licensed as GPLv3, so it'd be compatible with Ansible's licensing in that respect.

Would you be interested in filing a pull request with your module & lookup code for inclusion? I'd be happy to do the reviews and help it be included. 

Cheers,
Ryan

**Comment by pjodouin** [comment]
Hi Ryan,

I actually did submit a PR a while ago but it stagnated for a while, I got
really busy and just closed it.

I am more than willing to give this another try.

Cheers,
Pierre

Pierre Jodouin
*Cloud Solutions Architect*
*Innovative Virtual Computing Solutions Inc.*
*613.406.7008*


<https://ca.linkedin.com/pub/pierre-jodouin/1a/3b3/a79>

**Comment by pjodouin** [comment]
Hey @ryansb,

Let's start with my repo, get that cleaned up after which I will submit a PR for inclusion in extras.  Thanks!

**Comment by pjodouin** [comment]
@ryansb Just wondering if this is going to move.  Would you prefer if I just reissued my original PR?  Thanks.

**Comment by ryansb** [comment]
Yeah, it is - I'm working on updating your playbook samples (since you removed the `type=alias` among other things) and I'll file a PR.

**Comment by ryansb** [comment]
Hi, the first of these PR's has been opened (for `lambda` and `lambda_alias`)

**Comment by ryansb** [comment]
Another one for lambda_facts https://github.com/ansible/ansible-modules-extras/pull/2874

**Comment by michael-dev2rights** [comment]
I put in pull request #20890 which is basically adding lambda_policy with minor edits of meta-data.  @ryansb @pjodouin comments appreciated.
